tracks/ruby/exercises/hamming/README.md

```diff
@@ -1,30 +1,37 @@
 # Hamming
 
-Hamming is an exercise focused on iteration. It's is composed of two
-sub-problems: how to **iterate** through two collections, and how to **count**
+Hamming is composed of two sub-problems: how to **iterate** through two collections, and how to **count**
 based on a condition.
 
-The `Enumerable` module contains fantastic tools to solve these problems but
-many students are unfamiliar with it. This is a great exercise to introduce them
+The `Enumerable` module contains fantastic tools to solve these problems. This is a great exercise to introduce students
 to some of its features.
 
 ## Reasonable Solutions
 
-There are three strategies for the **iteration**. In my personal order of
-preference they are:
-
-1. Iterating through both strings simultaneously via `Enumerable#zip` (avoids
-   the need for indices)
-2. Iterating through indices directly using a `Range`, `Integer#upto`, or
-   `Integer#times`.
-3. A hybrid approach where you iterate through one of the strings while also
-   tracking the index to find the equivalent character in the other string. Uses
-`Enumerable#each_with_index`.
+There are three strategies for the **iteration**. In descending order of popularity with mentors:
+ 
+Hybrid approach 
+1. Iterating through both strings simultaneously via `Enumerable#zip`: 👍 avoids indices entirely, 👎 needs to convert the string to an array
+2. Iterating explicit indices using a `Range`, `Integer#upto`, or `Integer#times`: 👍 uses string directly, no array conversion needed, 👎 requires indices
+3. A hybrid approach with `Enumerable#each_with_index`, iterating over one of the strings while also
+   tracking the index to find the equivalent character in the other string. Uses `Enumerable#each_with_index`: 👍 more intuitive? students seem to pick this often, 👎 requires both indices and converting to arrays so it's kind of the worst of both worlds from 1 and 2 😛
```

Comment:
I wonder if we're packing a little too much information in this ordered list?  Would it make sense to move the analysis of pros/cons to a paragraph above each example? Or do you think it's better to keep it all together in this section here?

Reply:
Maybe we do pack too much here. Let me try something else....